app/views/ncr/proposals/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <h4>National Capital Region</h4>
   <h3>Purchase Card Approval Request</h3>
 
-  <%= form_for @proposal_form do |f| %>
+  <%= form_for @proposal_form, url: @form_url, method: @form_method do |f| %>
```

Comment:
I'm gonna take a crack at this in the morning – will time box to an hour and submit a PR on top of this one. If I can't get it, will go ahead and merge.